AuthorizationServiceApi.js

```diff
@@ -108,19 +120,18 @@
 					let encryptedKeyAsBuffer = encryptor.update(contentKeyAsBuffer);
 					encryptedKeyAsBuffer = Buffer.concat([encryptedKeyAsBuffer, encryptor.final()]);
 
-					message.keys.push({
+					message.content_keys_source.inline.push({
 						"id": key.keyId,
-						"encrypted_key": encryptedKeyAsBuffer.toString("base64")
+						"encrypted_key": encryptedKeyAsBuffer.toString("base64"),
+						"usage_policy": "Policy A"
 					});
 				});
 
 				// For detailed information about these fields, refer to Axinom DRM documentation.
 				let envelope = {
 					"version": 1,
 					"com_key_id": secrets.communicationKeyId,
-					"message": message,
-					"begin_date": validFrom.toISOString(),
-					"expiration_date": validTo.toISOString()
```

Comment:
They were examples on how to set the validity period for the Axinom DRM License Server Message (it envelopes the actual message, which is the Entitlement Message). These should be put back.